public/home.js

```diff
@@ -1,6 +1,41 @@
 $(document).ready(function(){
 
+  var APIaction = function(){
```

Comment:
As you are only going to have one instance of this class, it's better to write it as an object:

```js
var APIaction = {
  logOut: function(){
    $.ajax({
      type: 'DELETE',
      url: '/sessions',
      success: function(response){
        console.log(response)
        window.location.href = '/';
      }
    })
  }
};
```